ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -81,6 +82,7 @@ - (void)updateMap
         [_map.attributionButton setHidden:_attributionButton];
         [_map.logoView setHidden:_logo];
         [_map.compassView setHidden:_compass];
+        _map.contentInset = _inset;
```

Comment:
I’d mildly recommend setting this before setting `userTrackingMode`, just to reduce the likelihood of edge cases, since both properties can cause the map to move.